requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
@chouseknecht is there a place on the roadmap we could put this with a link to the work we have done? Where does that roadmap live?